anomalib/post_processing/post_process.py

```diff
@@ -5,13 +5,21 @@
 
 
 import math
+from enum import Enum
 from typing import Optional, Tuple
 
 import cv2
 import numpy as np
 from skimage import morphology
 
 
+class ThresholdMethod(str, Enum):
+    """Threshold method to apply post-processing to the output predictions."""
+
+    ADAPTIVE = "adaptive"
+    FIXED = "fixed"
```

Comment:
Do you think maybe `manual` would be a better name instead of `fixed`? As in manual threshold vs. adaptive threshold. (Sorry for being annoying, I realize I was the one who proposed fixed initially).

Reply:
I'm fine with `manual`. @ashwinvaidya17, do you have any preference?

Reply:
Works for me too